core/unit_test/TestMathematicalFunctions.hpp

```diff
@@ -394,10 +395,12 @@ DEFINE_UNARY_FUNCTION_EVAL(log2, 2);
 DEFINE_UNARY_FUNCTION_EVAL(log1p, 2);
 #endif
 
-#ifndef KOKKOS_MATHEMATICAL_FUNCTIONS_SKIP_1
+#ifndef KOKKOS_MATHEMATICAL_FUNCTIONS_SKIP_2
```

Comment:
Can you comment about what you did here (on GH). You just randomly moved tests from on exe to the other or was there some reasoning behind it?

Reply:
Monkey do monkey see.